src/control/server/storage/scm/provider.go

```diff
@@ -549,25 +551,33 @@ func (p *Provider) CheckFormat(req FormatRequest) (*FormatResponse, error) {
 		return nil, errors.Wrapf(err, "failed to check if %s is mounted", req.Mountpoint)
 	}
 	if isMounted {
+		res.Mounted = true
 		return res, nil
 	}
 
-	if req.Dcpm != nil {
-		fsType, err := p.sys.Getfs(req.Dcpm.Device)
-		if err != nil {
-			if os.IsNotExist(errors.Cause(err)) {
-				return nil, errors.Wrap(FaultFormatMissingDevice, req.Dcpm.Device)
-			}
-			return nil, errors.Wrapf(err, "failed to check if %s is formatted", req.Dcpm.Device)
-		}
+	if req.Dcpm == nil {
+		// ramdisk
+		res.Formatted = false
```

Comment:
Isn't a ramdisk by default formatted as tmpfs? I'm guessing there's a reason why we'd say it wasn't, but it would be nice to have a comment as to why.